tasks/freeradius.yml

```diff
@@ -1,8 +1,17 @@
 ---
+- name: Add freeradius repository into sources list
+  ansible.builtin.apt_repository:
+    repo: deb http://packages.networkradius.com/freeradius-3.0/debian/bullseye bullseye main
```

Comment:
In this do we have to check the ansible os family first and then add the apt repository?

Reply:
correct